### ISSUE TITLE
Consider to merge MathNet.Numerics.FSharp into MathNet.Numerics

### ISSUE THREAD

**Issue by cdrnet** [mathnet/mathnet-numerics#496]
Advantages:
- Simpler usage from F# (single package only)
- Freedom to implement certain functionality in F# yet still have it available for all users

Drawbacks:
- Dependency on Fsharp.Core NuGet package for all users

**Comment by Huqin-China** [comment]
yes!it's acceptable to add fsharpcore since it's cross-platform.By the way,net standard2.0preview is released

**Comment by cdrnet** [comment]
Yes, I think it would not be a blocking issue.

**Comment by iamcarbon** [comment]
Would this mean re-targeting everything to .NETSTANDARD?

**Comment by cdrnet** [comment]
It would indeed mean we'd have to reduce the supported targets to those supported by Fsharp.Core (which has started to focus on .Net Standard and newer .Net Framework only); unless we'd split/ifdef, so only those supporting Fsharp.Core would depend on Fsharp.Core and include those features.
